static/js/components/ProgressWidget_test.js

```diff
@@ -32,7 +32,9 @@ describe('ProgressWidget', () => {
     ));
     const wrapper = shallow(<ProgressWidget program={program}/>);
     const totalCourses = program.courses.length;
-    const passedCourses = program.courses.find(course => course.status === STATUS_PASSED);
+    const passedCourses = program.courses.filter(
+      course => course.runs.length > 0 && course.runs[0].status === STATUS_PASSED
+    );
     const totalPassedCourses = passedCourses ? passedCourses.length : 0;
```

Comment:
There shouldn't be any case where `passedCourses` is not a list, so we don't need the trinary expression here